monitor_ingest/actions/data_clean_csv_action.py

```diff
@@ -0,0 +1,102 @@
+import json
```

Comment:
From a naming perspective, a better name would be `data_clean_csv` you can leave out the _action

Reply:
updated